src/main/java/de/htw/basketmicroservice/port/user/BasketController.java

```diff
@@ -23,6 +23,15 @@ public BasketDTO getBasket(@PathVariable UUID basketId) {
 
     @DeleteMapping("baskets/{basketId}/items/{itemId}")
     public BasketDTO removeItem(@PathVariable UUID basketId, @PathVariable UUID itemId) {
-        return basketService.removeBasketItem(basketId, itemId);
+        basketService.removeBasketItem(basketId, itemId);
+        return basketService.getBasket(basketId);
     }
+
+    @PutMapping("baskets/{basketId}/items/{itemId}")
+    public BasketDTO changeItemAmount(
+            @PathVariable UUID basketId, @PathVariable UUID itemId, @RequestBody Integer amount) {
```

Comment:
consistent naming of attributes: rename amount to quantity -> BasketDTO JSON has quantity attirbute and basket POST request has also quantity attribute in request body

Reply:
done